packages/ui/components/localize/src/date/parseDate.js

```diff
@@ -63,12 +64,16 @@ export function parseDate(dateString, options) {
   }
 
   const [year, month, day] = parsedString.split('/').map(Number);
+  const parsedYear = parsedString.split('/')[0];
```

Comment:
Maybe keep the same way of writing as the line above.

```suggestion
  const [yearString] = parsedString.split('/');
```